src/deluge/model/clip/instrument_clip.cpp

```diff
@@ -96,10 +96,10 @@ InstrumentClip::InstrumentClip(Song* song) : Clip(CLIP_TYPE_INSTRUMENT) {
 
```

Comment:
updated global enums used for initializing the automation view instrument clip variables